.mocharc.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   timeout: 30000,
   ui: 'bdd',
   extension: 'js',
-  require: 'node_modules/mocha-expect-snapshot',
+  require: 'node_modules/mocha-expect-snapshot/index.js',
```

Comment:
```suggestion
  require: 'mocha-expect-snapshot',
```

I've added node_modules, it was required due to an outdated mocha installed globally in my machine.
Spotted later my error.

Reply:
ok